lib/class-wp-theme-json-resolver.php

```diff
@@ -311,15 +311,20 @@ public static function get_theme_data( $theme_support_data = array() ) {
 	private static function get_user_data_from_custom_post_type( $should_create_cpt = false, $post_status_filter = array( 'publish' ) ) {
 		$user_cpt         = array();
 		$post_type_filter = 'wp_global_styles';
-		$post_name_filter = 'wp-global-styles-' . urlencode( wp_get_theme()->get_stylesheet() );
```

Comment:
Braindump: the result of removing this is that the CPT name will inherit the title's, which is "Custom Styles", so we end up with CPTs with names (and guids) such as `custom-styles`, `custom-styles-2`, `custom-styles-3`, etc. Wondering if we could retain this or even have `custom-styles-<theme>` as a name.

Not that it matters much for our use as we'll have the taxonomy. However, I wonder what happens if/when we reach the last `guid` number. Does it start over from 0? At that point, is it ok that there are two CPTs with `custom-styles-2` names? Those sort of situations.